spec/tests/check.fmf

```diff
@@ -14,7 +14,17 @@ description: |
     panic detection, core dump collection or collection of system
     logs.
 
-    To control checks for a given test, use L1 ``check`` key.
+    The following checks are currently available:
+
+    dmesg
+        check the kernel righ buffer and store its contents
+
+    avc
+        check for unexpected SELinux AVC denials, gather details
```

Comment:
Is the 'unexpected' word necessary? Technically speaking they are expected if you test on broken version.

IMO they are just SELinux AVC denials,

Reply:
Yeah, makes sense, fixed in 6bdcc459.